content/post/career-ladders-part-3.md

```diff
@@ -0,0 +1,57 @@
+- Does promotion require demonstrating 100% of the competencies laid out for each level, or is there room for folks to specialize and exceed expectations in one area while not meeting the threshold in another?
+- Does your company promote people they believe have the ability to achieve at the next level, or must someone have already demonstrated that level of work for 3 or 6 months before they are eligible for promotion?
+- Is there a fixed promotion schedule or is it ad-hoc?
+- How do you get nominated for a promotion? Is it self-driven or manager-driven? Do you need to create a promo portfolio, or is it less structured than that?
+- How does level affect salary?
+- How does your team's ladder fits into the overall company approach to leveling and salaries, if there is one?
+Answer any other questions that come up, and make sure your team understands that this is a living document which will likely change over time.
+
+### Growth Conversations
+Have regular, growth-focused conversations with your team members. The cadence and content may vary, but I’ll outline what has worked for me.
+- When someone joins my team, I have a general career conversation with them to understand where they want to go and how their current position can help them get there. Occasionally I learn, “I don’t actually want to be a Senior Analyst, I want to become a Product Manager.” If so, I do what I can to help them work on developing skills and getting involved with projects that will help them make that transition.
+- For folks who do want a promotion on my team, we check in on the career ladder roughly twice a year. The team member and I go through the ladder separately to benchmark where we believe they are in regards to each competency. Doing this assessment independently and then comparing helps highlight areas of misalignment, which may be due to lack of clarity in the job ladder, a lack of self-awareness from your team member, or you not understanding their work or experience. Resolving that misalignment makes the next conversation, about growth areas, more focused and productive.
+- Armed with those benchmarks, we create a growth plan together by agreeing on a few areas where they haven't yet been able to demonstrate the skills or proficiencies for the next level, and they want to focus on improvement. Then we identify concrete projects and actions that will give them an opportunity to build and showcase their skills over the next several months. If they have any ongoing responsibilities or projects that don't align with these growth areas, determine what will happen to those projects - can they be handed to someone who will get more out of them? Can you give them a bigger role on those projects, so that there is still growth opportunity? It takes concerted effort to make space for growth while ensuring all of the team's commitments are met.
```

Comment:
This part is so hard as a manager, especially the last sentence. How do you trade-off individual's growth vs meeting aggressive team goals?